data/special_pointers.asm

```diff
@@ -133,7 +133,7 @@ SpecialsPointers:: ; c029
 	add_special Function1700ba
 	add_special Function170114
 	add_special BattleTowerBattle
-	add_special UnusedSpecial_Function1704e1
+	add_special Function1704e1
```

Comment:
Why did you remove ``Unused`` here?

Anyway wouldn't it be preferable if unused specials and predefs kept the ``Special_`` and ``Predef_`` keyword? Given that they are never referenced they won't look redundant anywhere and you can notice which special/prefef slots can be reused more "transparently". Not a big deal though.